src/components/side_panel/pivot/pivot_measure_display_panel/pivot_measure_display_panel_store.ts

```diff
@@ -0,0 +1,184 @@
+import { deepCopy } from "../../../../helpers";
+import { ALL_PERIODS } from "../../../../helpers/pivot/pivot_helpers";
+import { Get } from "../../../../store_engine";
+import { SpreadsheetStore } from "../../../../stores";
+import { _t } from "../../../../translation";
+import {
+  PivotCoreDefinition,
+  PivotCoreMeasure,
+  PivotDimension,
+  PivotMeasureDisplay,
+  PivotMeasureDisplayType,
+  UID,
+} from "../../../../types";
+
+export class PivotMeasureDisplayPanelStore extends SpreadsheetStore {
+  mutators = [
+    "cancelMeasureDisplayEdition",
+    "updateMeasureDisplayType",
+    "updateMeasureDisplayField",
+    "updateMeasureDisplayValue",
+  ] as const;
+
+  measureDisplay: PivotMeasureDisplay;
+
+  constructor(get: Get, private pivotId: UID, private initialMeasure: PivotCoreMeasure) {
+    super(get);
+    this.measureDisplay = initialMeasure.display || { type: "no_calculations" };
+  }
+
+  updateMeasureDisplayType(measureDisplayType: PivotMeasureDisplayType) {
+    this.updatePivotMeasureDisplay(
+      this.getMeasureDisplay(
+        measureDisplayType,
+        this.measureDisplay.fieldNameWithGranularity,
+        this.measureDisplay.value
+      )
+    );
+  }
+
+  updateMeasureDisplayField(fieldNameWithGranularity: string) {
+    this.updatePivotMeasureDisplay(
+      this.getMeasureDisplay(
+        this.measureDisplay.type,
+        fieldNameWithGranularity,
+        this.measureDisplay.value
+      )
+    );
+  }
+
+  updateMeasureDisplayValue(value: string) {
+    this.updatePivotMeasureDisplay(
+      this.getMeasureDisplay(
+        this.measureDisplay.type,
+        this.measureDisplay.fieldNameWithGranularity,
+        value
+      )
+    );
+  }
+
+  private updatePivotMeasureDisplay(newDisplay: PivotMeasureDisplay) {
+    const pivotDefinition = deepCopy(this.model.getters.getPivotCoreDefinition(this.pivotId));
+    const measureIndex = this.getMeasureIndex(this.initialMeasure.id, pivotDefinition);
+    const newMeasure = { ...pivotDefinition.measures[measureIndex], display: newDisplay };
+    pivotDefinition.measures[measureIndex] = newMeasure;
+    const result = this.model.dispatch("UPDATE_PIVOT", {
```

Comment:
They are not doing much yeah, but we still need buttons to navigate back to the previous panel. What do you propose instead ? Doing everyhing in a dialog instead of another side panel ?

Reply:
I would say that a simple "back" button would do the trick but I'm not sure it's better. The side panel is great as we can see the live change. Let's keep it this way at the moment I guess